PixelHunter1995/SceneParser.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Why do we need all this?

Reply:
Visual Studio just magically added this to my file when I create a new class... I can try removing it and see what happens.

Reply:
Det ante mig!

Reply:
Seems most of that was completely unnecessary, so I removed it.